Segment-Facebook-App-Events.podspec

```diff
@@ -21,8 +21,8 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.0'
-  s.dependency 'FBSDKCoreKit', '~> 4.15'
+  s.dependency 'Analytics'
```

Comment:
hey @OllaAshour, echoing brie's comment on the Amplitude PR - can we specify the target version here? thanks!